packages/ffmpeg/package.json

```diff
@@ -25,10 +25,9 @@
     "@types/fluent-ffmpeg": "^2.1.21"
   },
   "dependencies": {
-    "@ffmpeg-installer/ffmpeg": "^1.1.0",
-    "@ffprobe-installer/ffprobe": "^2.0.0",
-    "fluent-ffmpeg": "^2.1.2",
     "@motion-canvas/core": "^3.16.0",
-    "@motion-canvas/vite-plugin": "^3.15.1"
+    "@motion-canvas/vite-plugin": "^3.15.1",
+    "ffmpeg-ffprobe-static": "^6.1.1-rc.5",
```

Comment:
What's the reason for this change? Is something wrong with `@ffmpeg-installer`?

Reply:
I added the explanation to my second commits description:
```
To use a few ffmpeg audio filters, I had to update the version of ffmpeg.
The @ffmpeg-installer/ffmpeg package has a wildy outdated version of
ffmpeg, so switching to ffmpeg-ffprobe-static was needed.
```
Specifically, turning off normalization on the `amix` filter wasn't possible on the older version. The default normalization produced unexpected fluctuations in gain for me when a lot of sounds are played in quick succession

Reply:
I see. There may be a problem with licensing tho. Both ffmpeg and `@ffmpeg-installer` use LGPL licenses which are compatible with MIT, however `ffmpeg-ffprobe-static` uses GPL which may require us to switch to GPL as well. I think changing only the license of our `ffmpeg` package would be enough since other pacakges don't realy on it but I'll need to look into that.
Some time ago I proposed switching to GPL entirely #1015 but the community reception was a bit mixed.

Reply:
Ah, I should've checked the license.

Another option would be to include a fork of `@ffmpeg-installer` in the ffmpeg package with updated binaries, but that depends on how easy that would be to do.

Reply:
looking back at ffmpeg-installer:
https://github.com/search?q=repo%3Akribblo%2Fnode-ffmpeg-installer%20license&type=code
The overall package indeed uses an LGPL license but the inner platform-specific packages already use GPLv3. Wouldn't this already be problematic then?

Also, parts of FFmpeg use the GPL license instead of LGPL: https://www.ffmpeg.org/legal.html, are we sure we aren't using an unfreeze version of FFmpeg?

Given that FFmpeg itself has parts that already fall under the GPL license, I wouldn't be against changing the license of the ffmpeg renderer to GPL

Reply:
So I was wondering if a license change would even be needed, because you could say motion canvas only "links" to ffmpeg, it doesn't embed it (npm would be the linker in this case)

But according to Gnu, linking to a GPL library still requires a "GPL-compatible license": https://www.gnu.org/licenses/gpl-faq.html#LinkingWithGPL
I found this using a stackoverflow answer: https://stackoverflow.com/a/5367158

~~another reason fot me to dislike gnu~~

Reply:
I did a quick test by running:
```ts
import {exec} from 'child_process';
exec(`${ffmpegPath} -h encoder=h264`, (error, stdout, stderr) => {
  console.log(stdout);
});
```
and the encoder used for h264 is libx264 (GPL licensed) so I think we're already cooked.
I say we just change the entire `ffmpeg` package to `GPLv3` and not worry about it.